docs/operator-guide/customize_deployment.md

```diff
@@ -0,0 +1,46 @@
+# Customize Deployment
+
+Sometimes additional steps are required during application deployment. With the customization of the KubeRocketCI deployment, it's possible to override the deployment logic, which can be useful, for example, if you need to spin up and configure a database before deploying the main application, remove old information from the environment, redeploy the application, and initiate post-deployment configuration, tests, and so on.
+This is where custom deployment comes into play. This page provides comprehensive guidelines on how to adjust the deployment logic to cater your needs.
+
+## Deploy Custom Pipeline
+
+To customize your deployment pipeline, follow the steps below:
+
```

Comment:
maybe we can provide scheme how we use chain tt, pipeline, task?

Reply:
Done